README.md

```diff
@@ -92,14 +126,52 @@ TF_LOG_PROVIDER=INFO terraform refresh
 TF_LOG_PROVIDER=INFO terraform show
 ```
 
-### Stop Tasks
+### Stop Task
 
 ```
 TF_LOG_PROVIDER=INFO terraform destroy
 ```
 
 This terminates the `machine` (if still running), downloads `output`, and removes the persistent `disk_size` storage.
 
+## How it Works
+
+This diagram may help to see what TPI does under-the-hood:
+
+```mermaid
+flowchart LR
+subgraph tpi [what TPI manages]
+direction LR
+    subgraph you [what you manage]
+        direction LR
+        A([Personal Computer])
+    end
+    B[("Cloud Storage (low cost)")]
+    C{{"Cloud instance scaler (zero cost)"}}
+    D[["Cloud (spot) Instance"]]
+    A ---> |create cloud storage| B
+    A --> |create cloud instance scaler| C
+    A ==> |upload script & workdir| B
+    A -.-> |"offline (lunch break)"| A
+    C -.-> |"(re)provision instance"| D
+    D ==> |run script| D
+    B <-.-> |persistent workdir cache| D
+    D ==> |script end,\nshutdown instance| B
+    D -.-> |outage| C
+    B ==> |download output| A
+end
+style you fill:#FFFFFF00,stroke:#13ADC7
+style tpi fill:#FFFFFF00,stroke:#FFFFFF00,stroke-width:0px
+style A fill:#13ADC7,stroke:#333333,color:#000000
+style B fill:#945DD5,stroke:#333333,color:#000000
+style D fill:#F46737,stroke:#333333,color:#000000
+style C fill:#7B61FF,stroke:#333333,color:#000000
+```
+
+## Future Plans
+
+TPI is a CLI tool bringing the power of bare-metal cloud to a bare-metal local laptop. We're working on more featureful and visual interfaces. We'd also like to have more native support for distributed (multi-instance) training, more data sync optimisations & options, and tighter ecosystem integration with tools such as [DVC](https://dvc.org).
+
 ## Help
 
 The [getting started guide](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) has some more information. In case of errors, extra debugging information is available using `TF_LOG_PROVIDER=DEBUG` instead of `INFO`.
```

Comment:
💅🏼 

```suggestion
The [Getting Started](https://registry.terraform.io/providers/iterative/iterative/latest/docs/guides/getting-started) guide has some more information. In case of errors, extra debugging information is available using `TF_LOG_PROVIDER=DEBUG` instead of `INFO`.
```